citesphere/src/main/java/edu/asu/diging/citesphere/web/user/dto/CitationStatusesData.java

```diff
@@ -0,0 +1,16 @@
+package edu.asu.diging.citesphere.web.user.dto;
+
+import java.util.List;
+
+public class CitationStatusesData {
+    private List<String> movedCitations;
+    private List<String> notMovedCitations;
+
+    public void setMovedCitations(List<String> movedCitationsList) {
+        this.movedCitations = movedCitationsList;
+    }
+
+    public void setNotMovedCitations(List<String> notMovedCitationsList) {
+        this.notMovedCitations = notMovedCitationsList;
+    }
+}
```

Comment:
if this class is never used outside the controller (and not meant to be used outside), I would make this an inner class.